### PR TITLE
DOC: How to take ownership

### LINKED ISSUE #2121: DOC: How to take over ownership
I want to make official rules / a process how people could take over the ownership of the Github repository / the PyPI account, especially when I'm not active (for whatever reason).

This should prevent issues in future, although I hope that we never need it.

### PR DESCRIPTION
**Pull request by MartinThoma**
This serves as an assurance to the community that pypdf will not die only if I become inactive.

Closes #2121

## For the Reviewer

The built docs are here: https://pypdf--2123.org.readthedocs.build/en/2123/meta/taking-ownership.html

### CODE REVIEW

**Comment by MartinThoma** [comment]
@Lucas-C @pubpub-zz @MasterOdin @exiledkingcc What do you think about this? Does it make sense? Is information missing?

**Review by Lucas-C — APPROVED** [review]

File: docs/meta/taking-ownership.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The resources needed for maintaining pypdf are:
+
+* PyPI: [pypdf](https://pypi.org/project/pypdf/) and [PyPDF2](https://pypi.org/project/PyPDF2/)
+* Github: [pypdf](https://github.com/py-pdf/pypdf) (the repository, not the organization)
+* ReadTheDocs: [pypdf](https://readthedocs.org/projects/pypdf/) and [PyPDF2](https://readthedocs.org/projects/pypdf2/)
+
+## When may somebody take ownership?
+
+**No activity in 180 days**: If I don't answer e-mails (info@martin-thoma.de)
```
Comment (Lucas-C) [inline docs/meta/taking-ownership.md:24]: You don't fear being spammed on that email address?

Instead, you could for example some basic email obfuscation like this:
```html
<a href="javascript:(() => { window.location.href = atob('bWFpbHRvOmluZm9AbWFydGluLXRob21hLmRl'); })()">
```

**Comment by Lucas-C** [comment]
> @Lucas-C @pubpub-zz @MasterOdin @exiledkingcc What do you think about this? Does it make sense? Is information missing?

It all makes sense and everything seems accurate!

**Comment by pubpub-zz** [comment]
Sounds fine. But I wish to have to test it 😅

**Review by MasterOdin — APPROVED** [review]

File: docs/meta/taking-ownership.md (hunk 1 above)
Comment (MartinThoma) [reply docs/meta/taking-ownership.md:24]: I'm getting spammed already as this email address is used everywhere 😅

**Comment by exiledkingcc** [comment]
i feel honoured by your trust. 
it's sounds good to me.

**Pull request merged**